### DISCUSSION TITLE
How to Find Latest Release for Windows?

### DISCUSSION THREAD

**Discussion by johntdavis84** [AlexandreRouma/SDRPlusPlus#773] [Q&A]
Hello,

I apologize for what is probably a fairly stupid question, but I'm stumped.

The GitHub latest release is 1.0.4, but I've also seen references to 1.0.5 and 1.0.6.
In fact, I'm trying to get the sever working, which means I need 1.0.5 on the Windows-side to get the client working, apparently.

But I cannot figure out how to download that.

Do I have to do the nightly releases? I'm a bit confused about how to just download a nightly binary without compiling something.

Thanks!

**EDIT:** I figured out how to download the nightly build, but can't seem to delete this thread. -_-

**Comment by AlexandreRouma** [comment]
https://www.sdrpp.org/nightly
it'll redirect you to the latest nightly, scroll down to artifacts, click on the version for windows
